Packs/OpenPhish/Integrations/OpenPhish_v2/OpenPhish_v2.yml

```diff
@@ -3,6 +3,11 @@ commonfields:
   id: OpenPhish_v2
   version: -1
 configuration:
+- defaultvalue: ""
```

Comment:
should not the default value be true?

Reply:
I think not, Since we want to keep the same behaviour as older version.
@Arsenikr What do you think?

Reply:
change/added params does not change the current instance at the customer as it is saved on the DB.